projects/laji/src/app/+observation/observation.component.ts

```diff
@@ -40,7 +41,7 @@ export class ObservationComponent extends AbstractObservation implements OnInit,
 
   ngOnInit() {
     this.observationFacade.emptyQuery = {
-      _coordinatesIntersection: 100
+      _coordinatesIntersection: environment.type === 'vir' ? 0 : 100
```

Comment:
Hmm... Could we put this logic into the `vir` env file instead of doing a hard check for the env name here? The hard check isn't really that bad, but the env variable -way would be more consistent with our existing code. Relying on an env file is more agnostic approach also.

So `projects/vir/src/environments/environment.ts` would have `observationCoordinatesIntersection` at here we would check pull the value from the env. Similar to how `observationMapOptions` is implemented in the env files.

Reply:
Maybe a more extendable solution would be to have `observationQuery` var instead of `observationCoordinatesIntersection`, and `this.observationFacade.emptyQuery` would spread it into the var (`{...bla, ...(env.observationQuery || {})}`)

Reply:
hmm well i already did the simple version of this, it's easy to extend if it is ever needed